doc-index-updater/overlays/non-prod/kustomization.yaml

```diff
@@ -1,21 +1,21 @@
 namespace: doc-index-updater
 
 resources:
-- ../../base
-- SealedSecret-basic-auth-creds.yaml
-- SealedSecret-logs-storage-creds.yaml
-- SealedSecret-redis-creds.yaml
-- SealedSecret-search-creds.yaml
-- SealedSecret-sentinel-creds.yaml
-- SealedSecret-service-bus-creds.yaml
-- SealedSecret-storage-creds.yaml
+  - ../../base
+  - SealedSecret-basic-auth-creds.yaml
+  - SealedSecret-redis-creds.yaml
+  - SealedSecret-search-creds.yaml
+  - SealedSecret-sentinel-creds.yaml
+  - SealedSecret-service-bus-creds.yaml
+  - SealedSecret-storage-creds.yaml
 patchesStrategicMerge:
-- deployment.yaml
-- ingress.yaml
-- lock-timeout.yaml
+  - deployment.yaml
+  - ingress.yaml
+  - lock-timeout.yaml
+  - authentication.yaml
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
-- digest: sha256:73d39213b8f587e1d1d85af0dee8d30a872ff931c2e2240be30d2018844d18dd
-  name: mhraproductsnonprodregistry.azurecr.io/products/doc-index-updater
+  - digest: sha256:bfbf9b1fd7c118725751b850a41613586943e9930807744de905438ad3ab52d6
```

Comment:
Should revert this before merging

Reply:
Actually, what you could do is do the kustomisation of authentication.yaml doing nothing and get that committed in kustomisation.yaml so that it doesn’t get conflicts

Then you can have the actual overlays of authentication.yaml sitting there as a PR for when you are ready.